### DISCUSSION TITLE
Latency test for Java

### DISCUSSION THREAD

**Discussion by ChandanGirish** [OpenDDS/OpenDDS#4337] [General]
Hello Team,

I was looking at performance latency tests for OpenDDS, is there any latency test performed with Java bindings? for instance from Java-Java or Java-C++?

**Comment by jrw972** [comment]
I'm not aware of anything.  You may want to comment on https://github.com/OpenDDS/OpenDDS/discussions/4257#discussioncomment-6927946

**Comment by jrw972** [comment]
The various Java to Java and C++ combinations are exercised in the Java Messenger test.  You could add code to this example or other examples that processes the `source_timestamp` to determine the one-way latency.  If you do add this code, please consider submitting a PR as this would be a welcome improvement.
